### PR TITLE
Update using-content-search-web-part-instead-of-content-query-web-par…

### PR DESCRIPTION
**Pull request by jon-frick**
…t-to-improve-perfo.md

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by jon-frick** [comment]
It looks like we’ve pulled the sync query capability. We may need to discard this pull request.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md [hunk 1]
```diff
@@ -57,7 +57,14 @@ Adding a Content Search Web Part is very similar to a regular Content Query Web
   
 ## Creating the right search query for your Content Search Web Part
 
-Once you have added a Content Search Web Part, you can refine the search and return the items you want. For detailed instructions on how to do this, see the section,  *"Display content by configuring an advanced query in a Content Search Web Part"*  in [Configure a Content Search Web Part in SharePoint](https://support.office.com/article/Configure-a-Content-Search-Web-Part-in-SharePoint-0dc16de1-dbe4-462b-babb-bf8338c36c9a).
+Once you have added a Content Search Web Part, you can refine the search and return the items you want. As you configure your webpart, pay attention to the loading behavior. There are two choices:
```
Comment (JohanFreelancer9) [inline Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md]: As you configure your **web part** pay attention to the loading behavior.

Please change as indicated.

File: Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md [hunk 2]
```diff
@@ -57,7 +57,14 @@ Adding a Content Search Web Part is very similar to a regular Content Query Web
...
+- **Sync option: Issue query from the server** - When the server issues the query, the results will be fetched and returned to the client as part of the initial page response. This ensures that the client is able to immediately present the search results to the user upon page delivery. However, this adds cost and complexity to the server rendering phase of the request. This will ultimately increase the time required to deliver the page. In the event that the search results take longer than expected to return, the entire page is blocked. The client will not be able to begin rendering the page until the results have been retrieved. On pages where the Content Search Web Part is not the primary source of data, this will result in a degraded user experience.
```
Comment (JohanFreelancer9) [inline Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md]: However, this adds cost and complexity to the server rendering phase of the **request, and** will ultimately increase the time required to deliver the page.

Please do change as indicated.

File: Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md [hunk 3]
```diff
@@ -57,7 +57,14 @@ Adding a Content Search Web Part is very similar to a regular Content Query Web
...
+- **Async option: Issue query from the browser** - Choosing this option will reduce the amount of work required by the server to deliver the page to the client, improving response times. This means that the client will be allowed to begin the work of rendering the page more quickly than under the Sync option. During the client rendering phase, the search query will be executed. This means that any delays retrieving the results will only impact the single webpart. Other controls will be allowed to render independently. This is beneficial on more complicated pages, as no single control blocks others. However, if the user is expecting to interact with or view the results from the search query, their experience may be degraded until the results arrive.
```
Comment (JohanFreelancer9) [inline Enterprise/using-content-search-web-part-instead-of-content-query-web-part-to-improve-perfo.md]: This means that any delays retrieving the results will only impact the single **web part**.

Please do change as indicated.

**Comment by mypil** [comment]
@jon-frick - Do you still want the content of this pull request (PR) to be merged? If not, you can close this PR. Thanks.

**Comment by jon-frick** [comment]
No. We can ignore this request. Not sure how to close it on my end. Can you handle it?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mypil** [comment]
@AndreaBarr - As discussed, can you possibly assist to close this PR as requested by @jon-frick? 

Thank you.

**Pull request closed without merging**